service-base/src/main/java/org/eclipse/hono/service/tenant/TenantAmqpEndpoint.java

```diff
@@ -32,7 +32,7 @@
 /**
  * An {@code AmqpEndpoint} for managing tenant information.
  * <p>
- * This endpoint implements Hono's <a href="https://www.eclipse.org/hono/api/TODO/">Tenant API</a>. It receives AMQP 1.0
+ * This endpoint implements Hono's <a href="https://www.eclipse.org/hono/api/Tenant-API/">Tenant API</a>. It receives AMQP 1.0
```

Comment:
must be `tenant-api`, uppercase doesn't work

Reply:
Right - locally the uppercase seems to be the only working one (???).
Anyway, I change it to lowercase.